src/lib/index.ts

```diff
@@ -25,8 +26,11 @@ export default class BundleChecker {
     this.originalCwd = process.cwd();
   }
 
-  // Refactor this, it is doing too much
-  public async compare(): Promise<IBundleCheckerReport> {
+  /*
+   * Refactor this, it is doing too much
+   * @deprecated This will be deleted soon. Please use `compare` instead
+   */
+  public async compareDeprecated(): Promise<string> {
```

Comment:
Can we simplify and remove `compareDeprecated`?

Reply:
that would be a separate pr

Reply:
Ok lets wait for the next pr